RELEASE.md

```diff
@@ -0,0 +1,52 @@
+# Release Procedure
+
+When creating a release, a few steps are necessary.
+
+## Housekeeping
+
+### Check repository Issues and Pull Requests
+
+Make sure there aren't any pending issues before tagging a new release.
+
+## Release commit
+
+### Change version in `Dockerfile`
+
+The image version specified in [`Dockerfile`](./Dockerfile) should be the same
+version as the one that's going to be tagged.
+
+### Create release commit
+
+This commit should contain only the above change, and be titled "Release
+version vX.Y.Z.".
+
+### Create tag
+
+The tag should be signed and created locally:
+
+```
+$ git tag -a -s vX.Y.Z
+```
+
+And its contents should be only the title: "vX.Y.Z".
+
+### Create release
+
+After pushing the resulting tag, a GitHub release [should be
+created](https://github.com/cnpem/epics-in-docker/releases/new). It should have
+the same title as the tag, and the description can use the "Generate release
+notes" feature, but special care should be taken to split the merged PRs into
+the "Breaking changes", "New features", and "Bug fixes" categories. The first
+section of the release description should be "Who needs this update" and
+describe the most relevant changes and package updates and what their effect
+is.
```

Comment:
How about keeping the list from #16 explicit here? It might make it easier to look after specific (breaking) changes in large change sets.

Reply:
That's a good idea, will add it.